.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/google-auth-library-ruby/issues
+  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+ruby
```

Comment:
It isn't the busiest tag, but should we direct people to https://stackoverflow.com/questions/tagged/google-auth-library-ruby ?

Reply:
fixed!